app/controllers/application_controller.rb

```diff
@@ -152,4 +153,10 @@ def with_format(format, &block)
     nil
   end
 
+  def set_cache_headers # https://jacopretorius.net/2014/01/force-page-to-reload-on-browser-back-in-rails.html
```

Comment:
It feels like this method could have a more descriptive name, like `disable_browser_caching`. `set_cache_headers` doesn't really convey what it does.